build/common.sh

```diff
@@ -351,8 +351,11 @@ function kube::build::destroy_container() {
 #    local -r version_major="${BASH_REMATCH[1]}"
 #    local -r version_minor="${BASH_REMATCH[2]}"
 #    local -r version_patch="${BASH_REMATCH[3]}"
+#    local -r version_extra="${BASH_REMATCH[4]}"
```

Comment:
Please implement these mappings inside this function, rather than leaving the client to implement the mappings.